File: python/prime.py
import math

max = 1000000

for num in range (3, max, 2) :
    prime = 1
    sqrt = int(math.sqrt(num))
    for i in range(3, sqrt, 2):
        if num%i == 0:
            prime =0
            break

    if prime == 1:
        print(num)

File: js/prime.js
const max = 1e6;

for (let num = 3; num < max; num += 2) {
    prime = true;
    sqrt = Math.sqrt(num);
    for (let i = 3; i <= sqrt; i += 2) {
        if (num % i == 0) {
            prime = false;
            break;
        }
    }
    if (prime) {
        console.log(num);
    }
}